app/views/users/show.html.erb

```diff
@@ -16,73 +16,79 @@
         <% if current_user.stores %>
-            <h3 class = "text-center">Admin for</h3>
-            <ul> <%= render partial: "layouts/admin_store", collection: @admin_stores %></ul>
+          <h3 class="text-center">Admin for</h3>
+          <ul> <%= render partial: "layouts/admin_store", collection: @admin_stores %></ul>
         <% end %>
       </div>
     </div>
   </div>
 
-<div class="container well transparent" id="order-history">
-  <div class="row">
-    <div class="col-md-4"><h2>Order History</h2></div>
-  </div>
-  <div class="row">
-    <div class="col-md-12">
-      <table class="table table-striped table-hover">
-        <thead>
-        <tr>
-          <th>Date</th>
-          <th>Order #</th>
-          <th>Images Ordered</th>
-          <th>Total</th>
-        </tr>
-        </thead>
-        <tbody>
-        <% @orders.each do |order| %>
-          <tr>
-            <td><%= order.created_at.strftime("%m/%d/%Y") %></td>
-            <td id="order-number-<%= order.id %>"><%= link_to order.id, order %></td>
-            <td id="order-image-count-<%= order.id %>"><%= order.order_items.count %></td>
-            <td id="order-total-<%= order.id %>"><%= dollars(order.total) %></td>
-          </tr>
-        <% end %>
-        </tbody>
-      </table>
-    </div>
-  </div>
-</div>
-
-  <div class="container well transparent"</div>
+  <div class="container well transparent" id="order-history">
     <div class="row">
-      <div class="col-md-4"><h2>Download Center</h2></div>
+      <div class="col-md-4"><h2>Order History</h2></div>
     </div>
     <div class="row">
       <div class="col-md-12">
-        <div>
-          <% @photos.each do |photo| %>
-            <%= link_to image_tag(photo.image_url, size: "100"), store_photo_path(photo.store.slug, photo) %>
+        <table class="table table-striped table-hover">
+          <thead>
+          <tr>
+            <th>Date</th>
+            <th>Order #</th>
+            <th>Images Ordered</th>
+            <th>Total</th>
+          </tr>
+          </thead>
+          <tbody>
+          <% @orders.each do |order| %>
+            <tr>
+              <td><%= order.created_at.strftime("%m/%d/%Y") %></td>
+              <td id="order-number-<%= order.id %>"><%= link_to order.id, order %></td>
+              <td id="order-image-count-<%= order.id %>"><%= order.order_items.count %></td>
+              <td id="order-total-<%= order.id %>"><%= dollars(order.total) %></td>
+              <td id="order-show-<%= order.id %>"><%= link_to "View",
+                              order_path(order),
+                              method: :get,
+                              class:  "btn btn-primary btn-sm inline" %>
+              </td>
+            </tr>
           <% end %>
-        </div>
+          </tbody>
+        </table>
       </div>
     </div>
   </div>
+
+  <div class="container well transparent"
+</div>
+<div class="row">
+  <div class="col-md-4"><h2>Download Center</h2></div>
+</div>c
```

Comment:
The indenting in this file is weird since there are several </div>s at the end all lined up. More importantly, though, there's a stray 'c' here on line 81. That's just a mistake, right?